composer.json

```diff
@@ -32,7 +32,8 @@
         "drift/http-kernel": "0.1.*, >=0.1.15",
         "laminas/laminas-diactoros": "^2",
         "vimeo/psalm": "^4",
-        "ext-fileinfo": "*"
+        "ext-fileinfo": "*",
```

Comment:
Could this be tagged as well with PHP version?

Reply:
done

Reply:
I have looked into the `ext-<lib>` versioning and it seems that it should remain on  `*` as suggested here: https://getcomposer.org/doc/01-basic-usage.md#platform-packages

it does make sense since the `ext-<libs>` are basically platform packages that are installed without composer.
I think we should change back both `ext-fileinfo` and `ext-curl` to `*`

@mmoreram WDTY?